modules/core/src/main/kotlin/com/metalbear/mirrord/MirrordConfigAPI.kt

```diff
@@ -103,19 +103,7 @@ class MirrordConfigAPI(private val service: MirrordProjectService) {
             return it.path
         }
 
-        if (!createIfMissing) {
-            return null
-        }
-
-        val config = createDefaultConfig()
-        service.notifier.notification(
-            "Created a mirrord default config",
-            NotificationType.WARNING
-        )
-            .withOpenFile(config)
-            .withDontShowAgain(MirrordSettingsState.NotificationId.DEFAULT_CONFIG_CREATED)
```

Comment:
If we're removing this, we should also remove the setting for disabling this NotificationId (it's still there in the settings dialog)